examples/bell-variational/README.md

```diff
@@ -34,7 +34,7 @@ This example consists of a very simple parametrized circuit with two variational
 
 ![circuit](images/circuit.png)
 
-Ideally, the circuit should prepare a maximally entangled state and set up a direction of measurment of 45 (225) degrees.
+Ideally, the circuit should prepare a maximally entangled state and set up a direction of measurement of 45 (225) in degrees.
```

Comment:
Thanks, I see that the angles printed by the script are [87, 43], maybe we should say something about that here?